rules/windows/execution_command_shell_started_by_svchost.toml

```diff
@@ -19,9 +19,15 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where event.type in ("start", "process_started") and
+process where event.type == "start" and
   process.parent.name : "svchost.exe" and process.name : "cmd.exe" and 
-  not (process.pe.original_file_name == "Cmd.Exe" and process.args : "?:\\Program Files\\Npcap\\CheckStatus.bat??")
+  not (process.pe.original_file_name : "cmd.exe" and process.args : (
+    "??:\\Program Files\\Npcap\\CheckStatus.bat?",
```

Comment:
Question - What does "??" allow for. I understand "?" is a wildcard but doubled I am not familiar with.

Reply:
The second one covers the disk letter